toolkit/tools/imagegen/installutils/installutils.go

```diff
@@ -53,19 +56,25 @@ type PackageList struct {
 // - mountPointDevPathMap is a map of mountpoint to partition device path
 // - mountPointToFsTypeMap is a map of mountpoint to filesystem type
 // - mountPointToMountArgsMap is a map of mountpoint to mount arguments to be passed on a call to mount
-func CreateMountPointPartitionMap(partDevPathMap, partIDToFsTypeMap map[string]string, config configuration.SystemConfig) (mountPointDevPathMap, mountPointToFsTypeMap, mountPointToMountArgsMap map[string]string) {
+// - diffDiskBuild is a flag that denotes whether this is a diffdisk build or not
+func CreateMountPointPartitionMap(partDevPathMap, partIDToFsTypeMap map[string]string, config configuration.SystemConfig) (mountPointDevPathMap, mountPointToFsTypeMap, mountPointToMountArgsMap map[string]string, diffDiskBuild bool) {
 	mountPointDevPathMap = make(map[string]string)
 	mountPointToFsTypeMap = make(map[string]string)
 	mountPointToMountArgsMap = make(map[string]string)
+	diffDiskBuild = false
```

Comment:
minor: no need to set this value to false, in golang all variabled are zeroed out by default.

Reply:
Fixed.

---
In reply to: [573074733](https://github.com/microsoft/CBL-Mariner/pull/611#discussion_r573074733) [](ancestors = 573074733)